gulpfile.js

```diff
@@ -0,0 +1,232 @@
+// This file is in charge of the build process:
```

Comment:
I do not know how to do a file comment so I comment the first line. In OpenPaaS we used Grunt for the task runner, if you can you should use Grunt instead of Gulp to have the same lib than us.